src/validation/overseas.name.validation.ts

```diff
@@ -0,0 +1,7 @@
+import { body } from "express-validator";
+import { EntityNameKey } from "../model/data.types.model";
+import { checkEntityName } from "./custom.validation";
+
+export const overseasName = [
+  body(EntityNameKey).custom((value) => checkEntityName(value))
```

Comment:
I actually prefer 'entityName' as the term/name to use for all the code artefacts when referring to the field on this new screen - is it too much work to refactor and use this across the board in the web app?

If yes, then we should then really change this function name to be consistent and call it `checkOverseasName`.

Reply:
Could you please be more specific